install_config/default_imagestreams_templates.adoc

```diff
@@ -0,0 +1,420 @@
+= Default Image Streams and Templates
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+// tag::firststeps[]
+
+== Overview
+
+ifdef::openshift-enterprise[]
+Your OpenShift installation includes a useful set of Red Hat-provided
+link:../architecture/core_concepts/builds_and_image_streams.html#image-streams[image
+streams] and link:../architecture/core_concepts/templates.html[templates] to
+make it easy for developers to create new applications. By default, the
+link:../install_config/install/quick_install.html[quick installation] and
+link:../install_config/install/advanced_install.html[advanced installation]
+methods automatically create these sets in the *openshift* project, which is a
+default global project to which all users have view access.
+endif::[]
+ifdef::openshift-origin[]
+You can populate your OpenShift installation with a useful set of Red
+Hat-provided
+link:../../architecture/core_concepts/builds_and_image_streams.html#image-streams[image
+streams] and link:../../architecture/core_concepts/templates.html[templates] to
+make it easy for developers to create new applications. By default, the
+link:advanced_install.html[advanced installation] method automatically creates
+these sets in the *openshift* project, which is a default project to which all
+users have view access.
+endif::[]
+
+ifdef::openshift-enterprise[]
+The core set of image streams and templates are provided and supported by Red Hat
+with an active OpenShift Enterprise subscription for the following technologies:
+
+[horizontal]
+Languages::
+- link:../using_images/s2i_images/nodejs.html[Node.js]
+- link:../using_images/s2i_images/perl.html[Perl]
+- link:../using_images/s2i_images/php.html[PHP]
+- link:../using_images/s2i_images/python.html[Python]
+- link:../using_images/s2i_images/ruby.html[Ruby]
+Database::
+- link:../using_images/db_images/mongodb.html[MongoDB]
+- link:../using_images/db_images/mysql.html[MySQL]
+- link:../using_images/db_images/postgresql.html[PostgreSQL]
+Other Services::
+- link:../using_images/other_images/jenkins.html[Jenkins]
+endif::[]
+
+ifdef::openshift-origin[]
+Image streams and templates are provided for the following technologies:
+
+[horizontal]
+Languages::
+- link:../../using_images/s2i_images/nodejs.html[Node.js]
+- link:../../using_images/s2i_images/perl.html[Perl]
+- link:../../using_images/s2i_images/php.html[PHP]
+- link:../../using_images/s2i_images/python.html[Python]
+- link:../../using_images/s2i_images/ruby.html[Ruby]
```

Comment:
probably not worth fixing here, but we have a Wildfly image too that we really ought to get documented:
https://github.com/openshift/sti-wildfly/